tasks/install/1.x.yml

```diff
@@ -28,3 +28,13 @@
   environment:
     JAVA_HOME: /usr/share/java
   with_items: "{{ sansible_elasticsearch_plugins }}"
+
+- name: Set ElasticSearch Startup Timeout
+  become: yes
+  lineinfile:
+    path: /etc/init.d/elasticsearch
+    regexp: "timeout=10"
+    line: "                timeout={{ sansible_elasticsearch_startup_timeout }}"
+  changed_when: false
```

Comment:
Remove this; it should change when the setting has been changed, and with this setting, it would never notify elasticsearch to restart.